sdk/python/feast/feature_table.py

```diff
@@ -254,54 +267,50 @@ def _to_data_source(cls, data_source):
         Convert dict to data source.
```

Comment:
This method is mysterious. Can you please add documentation so that its clear what its purpose is. By looking at the name I would expect it to convert a feature table into a data source.

Reply:
Added more description and renamed the function.